src/lib/s3Uploader.js

```diff
@@ -24,6 +24,21 @@ if (process.env.VCAP_SERVICES) {
 }
 export const s3 = new S3(s3Config);
 
+export const getPresignedURL = (Key, Bucket = bucketName, s3Client = s3, Expires = 360) => {
+  let url;
+  try {
+    const params = {
+      Bucket,
+      Key,
+      Expires,
+    };
+    url = s3Client.getSignedUrl('getObject', params);
+  } catch (error) {
+    return error;
```

Comment:
I think we will want to allow any errors while generating the presigned url to bubble up to the caller. What I'm not sure of is how we want to handle it on the API side. First thought is instead of returning just the url returning an object that can contain an error message if the URL isn't able to be generated. I'm not sure of the support for a virtual field that is an object.

Reply:
Makes sense to bubble up the error. We could still use the try/catch to log the error, e.g. - before rethrowing.

Reply:
returning an object for a virtual field works fine, so we could return an error if it occurs. What should we do with it though?

Reply:
I imagine on the frontend we will check for the error and displays a message if there is one. But as far as the server is concerned returning the error object seems fine to me.

Reply:
So you are good with it how it is?